src/components/ZodiacSign/ZodiacSign.js

```diff
@@ -0,0 +1,15 @@
+import './ZodiacSign.css';
+
+export default function ZodiacSign(booger) {
+  return (
+    <div className="zodiac-sign">
+      <div className="zodiacs-container">
+        <img src={`${process.env.PUBLIC_URL}/images/${booger.name}.png`} />
```

Comment:
no need for the process.env.public_url.  it should work with the pathing from images.